payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/StripePaymentProcessor.kt

```diff
@@ -196,6 +209,35 @@ class StripePaymentProcessor : PaymentProcessor {
                 ProductInfo(product.delete().id)
             }
 
+    override fun addSource(customerId: String,
+                           sourceId: String,
+                           setDefault: Boolean): Either<PaymentError, SourceInfo> =
+            either("Failed to add source $sourceId to customer $customerId") {
```

Comment:
Very obscure control flow.  Please rewrite for readability (whole method)

Reply:
Have to disagree here, provided that you mean the `either()` call. The `either` wrapper i a `try catch` wrapper that will turn exception thrown on calls to Stripe into an `Either`. Used in all methods calling Stripe.